modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -0,0 +1,50 @@
+= OpenID Connect (OIDC)
+:toc: right
+:toclevel: 2
+:openid-connect-frontchannel-logout-url: https://openid.net/specs/openid-connect-frontchannel-1_0.html
+:openid-connect-url: https://openid.net/connect/
+:openid-config-url: http://localhost:3000/.well-known/openid-configuration
+
+== Introduction
+
+"OpenID Connect 1.0 is a simple identity layer on top of the OAuth 2.0 protocol. It allows Clients to verify the identity of the End-User based on the authentication performed by an Authorization Server, as well as to obtain basic profile information about the End-User in an interoperable and REST-like manner.
+OpenID Connect allows clients of all types, including Web-based, mobile, and JavaScript clients, to request and receive information about authenticated sessions and end-users. The specification suite is extensible, allowing participants to use optional features such as encryption of identity data, discovery of OpenID Providers, and session management, when it makes sense for them."
+-- {openid-connect-url}, "What is OpenID Connect?"
+
+=== Configure ownCloud
```

Comment:
Why do you have the headers a sub hierarchy of introduction, this should be imho `==` not `===`.

Reply:
Likely was a oversight.